.circleci/config.yml

```diff
@@ -1,111 +1,85 @@
 version: 2.1
-job_configuration: &job_configuration
-  docker:
-    - image: circleci/node:10-stretch
-  working_directory: ~/repo
-jobs:
-  build-and-test:
-    <<: *job_configuration
-    resource_class: xlarge
+
+orbs:
+  node: circleci/node@1.1.6
+
+executors:
+  node-executor:
+    docker:
+      - image: circleci/node:13-stretch
```

Comment:
should it be using `buster`? Super nit, was looking into this and the stable version currently is `buster`